source/core/src/main/com/csse3200/game/components/npc/DodgingComponent.java

```diff
@@ -30,6 +32,7 @@ public class DodgingComponent extends Component {
   private float dodgeSpeed = 1.75f; 
   private float originalSpeed; // Original entity vertical speed
   private PhysicsEngine physics;
+  private Random random = new Random();
```

Comment:
Random keeps popping up as an issue on SonarCloud, is there anything else that could be used instead?